server/text_generation_server/models/custom_modeling/flash_pali_gemma_modeling.py

```diff
@@ -92,7 +93,7 @@ def forward(
 
             # insert image features into input embeddings
             inputs_embeds[mask] = image_features.view(-1, image_features.shape[-1])
-
+        input_lengths = Seqlen(input_lengths=input_lengths)
```

Comment:
This currently breaks CUDA inference.

I haven't looked deeply yet, but the issue could be that the wrapping should not happen with CUDA Graphs enabled (see the flash causal LM implementation).

Also, I think we could do the wrapping in `VlmCausalLM.forward`, which would be nicer than doing it per model.

Reply:
agreed,update the PR